synctl

```diff
@@ -1020,6 +1021,10 @@ class SyntheticPoP(Base):
             print('no location')
         print('total:', len(pop_data))
 
+    def print_credentials(self, credentials):
+        print("CREDENTIALS")
```

Comment:
only one column to show for creds, I think we can remove the title `CREDENTIALS`. It is easy to confuse it with credentials. For example:
```
CREDENTIALS
BIN_PASS4
BIN_PASS3
BIN_PASS2
USER_NAME_TEST
BIN_PASS1
BIN_PASS5
```

Reply:
The list is unsorted, it is not easy to find if the list is very long. can you sort it alphabetically？